cmd/cronosd/config/prefix_mainnet.go

```diff
@@ -1,3 +1,4 @@
+//go:build !testnet
```

Comment:
should we keep only one of those?

Reply:
I'm not sure actually, it's added by `gofmt`.

Reply:
maybe you can try to remove 
```
// +build !testnet
```
and test if it works?

Reply:
I find the answer here, they are both useful:
https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives